plugins/search-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -168,10 +169,11 @@ export const SearchBarBase: ForwardRefExoticComponent<SearchBarBaseProps> =
               endAdornment: clearButton
                 ? clearButtonEndAdornment
                 : endAdornment,
+              ...InputProps,
             }}
             inputProps={{
               'aria-label': ariaLabel,
-              ...defaultInputProps,
+              ...inputProps,
```

Comment:
Changed the name from `defaultInputProps` to `inputProps` to avoid a clash with `InputProps` capital I.

Reply:
Since `inputProps` -> `defaultInputProps` would translate the same with `InputProps`.